tests/TestCase/Command/BakeMigrationDiffCommandTest.php

```diff
@@ -36,9 +36,34 @@ protected function getDumpSchema(Arguments $args)
     {
         $diffConfigFolder = Plugin::path('Migrations') . 'tests' . DS . 'comparisons' . DS . 'Diff' . DS;
         $diffDumpPath = $diffConfigFolder . 'schema-dump-test_comparisons_' . env('DB') . '.lock';
-        $file = unserialize(file_get_contents($diffDumpPath));
 
-        return $file;
+        return unserialize(file_get_contents($diffDumpPath));
+    }
+
+}
+class CustomSimpleBakeMigrationDiffCommand extends BakeMigrationDiffCommand
```

Comment:
We shouldnt put those here, but inside TestApp dir etc.

Reply:
yes , but only uses in this testCase , tracking it when is not here could be hard later

Reply:
This violates psr loading rules and a bunch of other things.
As such still usually better to follow this strictly. One class per file.

Reply:
and I no like to use custom class , but I not found better way

Reply:
if phpstan is not happy with this , I  can put it to TestApp dir

Reply:
classes in TestApp are free :)

Reply:
moved